airflow/www/extensions/init_views.py

```diff
@@ -156,6 +158,21 @@ def set_cors_headers_on_response(response):
     return response
 
 
+class CustomSessionInterface(SecureCookieSessionInterface):
```

Comment:
```suggestion
class CustomSessionInterface(SecureCookieSessionInterface):
```

I would prefer a better name on the class interface but I can't come up with any..